docs/monitoring/metrics.md

```diff
@@ -1276,9 +1276,17 @@ Metrics related to data exchange between task executors using netty network comm
     </tr>
     <tr>
       <td>fullRestarts</td>
-      <td>The total number of full restarts since this job was submitted.</td>
+      <td>
+        The total number of restarts since this job was submitted.
```

Comment:
What is the difference to `numberOfRestarts`? Should `fullRestarts` be eventually deprecated?

Reply:
Yes. It is superseded and we may deprecate it in another task, see FLINK-14643.

Reply:
Ok, actually we can already put a note in the documentation that `fullRestart` is deprecated.

Reply:
I have removed this hotfix and think we should do it in FLINK-14643 along with the deprecation of `fullRestart`.
The fix of this description is another task FLINK-14641, since it should be applied to 1.9 as well.